testing/integration-tests/src/frame/contracts.rs

```diff
@@ -220,6 +220,8 @@ async fn tx_call() {
         .contracts()
         .contract_info_of(&contract);
 
+    let info_addr_bytes = cxt.client().storage().address_bytes(&info_addr).unwrap();
```

Comment:
Slightly annoying to have to do this extra step now to generate the bytes from `info_addr_bytes` from `cxt.client().storage()` (and handle the possible `Error`)

Reply:
(One thing I would like to do anyway is improve the storage APIs so that you don't ever feel the need to generate the address when using Subxt APIs. Eg here, using `fetch_keys` below just simplifies this test because you are happy to consider just 10 of the keys. Using `iter()` to iterate them all might be more correct but a little more verbose becasue you can't do a the nice chaining; I wonder whethrI should tweak this test to use the "iter" API; I think I almost did when I first saw this)